tests/title-show-test.js

```diff
@@ -23,7 +23,8 @@ describeApplication('TitleShow', function() {
       this.server.create('identifier', { type: 1, subtype: 1, id: '978-0547928210' }),
       this.server.create('identifier', { type: 1, subtype: 1, id: '978-0547928203' }),
       this.server.create('identifier', { type: 1, subtype: 2, id: '978-0547928197' }),
-      this.server.create('identifier', { type: 1, subtype: 0, id: '978-0547928227' })
+      this.server.create('identifier', { type: 1, subtype: 0, id: '978-0547928227' }),
+      this.server.create('identifier', { type: 8, subtype: 49, id: 'someothertypeofid' })
```

Comment:
Was there meant to be a matching test for this?

Reply:
Yep, I totally meant to do that. Added.